internal/services/managed_environment/resource_managed_environment.go

```diff
@@ -284,11 +380,16 @@ func (r *ManagedEnvironmentResource) Update(ctx context.Context, req resource.Up
 				SuppressValidationEmails:       strconv.FormatBool(plan.SuppressValidationEmails.ValueBool()),
 				MakerOnboardingUrl:             plan.MakerOnboardingUrl.ValueString(),
 				MakerOnboardingMarkdown:        plan.MakerOnboardingMarkdown.ValueString(),
+				SolutionCheckerRuleOverrides:   "",
```

Comment:
are you sure this should be sent as "" to the backend? Is that how ppac does it?